implementations/rust/ockam/ockam_command/src/util/config/mod.rs

```diff
@@ -150,15 +140,6 @@ Otherwise your OS or OS configuration may not be supported!",
             .pid)
     }
 
-    /// Check whether another node has been registered with this API
-    /// port.  This doesn't catch all port collision errors, but will
-    /// get us most of the way there in terms of starting a new node.
-    pub fn port_is_used(&self, port: u16) -> bool {
-        let inner = self.inner.read().unwrap();
-
-        inner.nodes.iter().any(|(_, n)| n.port == port)
-    }
-
```

Comment:
Please don't remove this function even though it's currently not used. It is still needed in the future

Reply:
OK, I'll revert the visibility change of `OckamCommand` to avoid the unused warning.